аbout.html

```diff
@@ -1,41 +1,52 @@
 <!DOCTYPE html>
 <html>
-<head>
- <title>About</title>
-</head>
- <style>h1 {text-align: center;}</style>
- <style>h2 {text-align: center;}</style>
- 
-<body> 
-<header>
-  <nav>
-   <h1>HTML</h1>
-   <h2>About page</h2>
- <ul> 
-    <li><a href="https://www.youtube.com/watch?v=bYodg42FAd4&list=RDMMbYodg42FAd4&start_radio=1">Home</a></li>
-    <li><a href="./assets/about.jpg">About</a></li>
-    <li><a href="./contact page.html">Contact</a></li>
- </ul>
-  </nav>
-</header> 
-<main>
- <h3>About our company</h3>
- <section>
-   <button type="button">Learn more</button>
-</section>
-<section>
-  <p><img src="./assets/image4.jpg" alt="Testpic" width="350" height="300"></p>
- </section>
- <section>
-   <p>Contrary to popular belief, Lorem Ipsum is not simply random text. 
-    It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old.</p>
-   <p>He standard chunk of Lorem Ipsum used since the 1500s
-    is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by Cicero are also reproduced 
-    in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham.</p>
- </section>
-</main>
-<footer>
-    <p><b> &#169; All rights reserved </b></p>  
-  </footer>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <title>About</title>
+    <link rel="stylesheet" href="styles-about.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="./аbout.html">About</a></li>
+          <li><a href="./contact page.html">Contacts</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h1>About our company</h1>
+      <br />
+      <section><button type="button">Learn more</button></section>
+      <br />
+      <section>
+        <p>
+          <img
+            src="./assets/cobblehill.jpg"
+            alt="Testpic"
+            width="350"
+            height="300"
+          />
+        </p>
+      </section>
+      <section class="margin">
```

Comment:
Class name of the elements should describe their content and what they are representing